app/assets/stylesheets/components/_notice.scss

```diff
@@ -8,7 +8,8 @@
     padding: $gutter-half;
   }
 
-  h2 {
+  // scss-lint:disable SelectorFormat
```

Comment:
We can get rid of these selectors now, also the nesting is not necessary anymore